### PR TITLE
docs: announcement for the new SQL Editor

### PR DESCRIPTION
**Pull request by Candybase**
The blog post introduces 3 features: the new SQL Editor, data anonymization, and database access control.

### CODE REVIEW

**Review by tianzhou — CHANGES_REQUESTED** [review]
I have tweaked some content directly. The left items is to add the audit log content.

**Comment by adela-bytebase** [comment]
I can't view the image, don't know why ...

File: content/blog/the-sql-editor-for-developers-and-dbas.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+- [Data anonymization](#data-anonymization): The query result of sensitive columns will be masked when developers run a query in SQL Editor. It is applicable to all query types, such as subquery, JOIN, Common Table Expression (CTE).
+- [Collaborative sheets](#collaborative-sheets): DBAs and developers can save SQL statements as sheets to share with others in SQL Editor. Moreover, this release supports saving the uploaded SQL scripts as sheets to be used in issues to apply database changes, a single sheet can hold up to 100MB sized SQL statements.
+- Good user experience: This release presents SQL Editor with a fresh look and improves the code auto-completion.
+
+### Admin Mode
+
+When users want to connect to databases to run admin commands, they can do it in SQL Editor Admin mode with a DBA account like the screenshot below. You can take a look at the document [Admin Mode](/docs/sql-editor/admin-mode) to learn more details.
+![admin-mode](/static/blog/the-sql-editor-for-developers-and-dbasadmin-mode.webp)
```
Comment (tianzhou) [inline content/blog/the-sql-editor-for-developers-and-dbas.md:34]:
```suggestion
![admin-mode](/static/blog/the-sql-editor-for-developers-and-dbas/admin-mode.webp)
```

File: content/blog/the-sql-editor-for-developers-and-dbas.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+### Database-based Access Control
+
+When users want to allow developers to access some databases in the production environment, they can mark the production environment as a protected environment and configure an allowlist. You can get more details from the document [Database Access Control](https://www.bytebase.com/docs/administration/database-access-control).
+![access-control](/static/blog/the-sql-editor-for-developers-and-dbasaccess-control.webp)
```
Comment (tianzhou) [inline content/blog/the-sql-editor-for-developers-and-dbas.md:39]:
```suggestion
![access-control](/static/blog/the-sql-editor-for-developers-and-dbas/access-control.webp)
```

File: content/blog/the-sql-editor-for-developers-and-dbas.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+### Data Anonymization
+
+When developers query tables containing sensitive columns in SQL Editor, the result of these columns will display as "**\*\***". You can learn the configuration details from the document [Anonymize Data](https://www.bytebase.com/docs/administration/anonymize-data).
+![anonymize-data](/static/blog/the-sql-editor-for-developers-and-dbasanonymize-data.webp)
```
Comment (tianzhou) [inline content/blog/the-sql-editor-for-developers-and-dbas.md:44]:
```suggestion
![anonymize-data](/static/blog/the-sql-editor-for-developers-and-dbas/anonymize-data.webp)
```

File: content/blog/the-sql-editor-for-developers-and-dbas.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+### Collaborative Sheets
+
+For commonly-used SQL statements, you can save them as sheets and set up a public SQL script library in your team. For large SQL scripts (up to 100M), you can use them in issues after uploading them as sheets (see the screenshot below). You can explore the document [Manage SQL Scripts with Sheet](https://www.bytebase.com/docs/sql-editor/manage-sql-scripts) to get additional details.
+![large-script](/static/blog/the-sql-editor-for-developers-and-dbaslarge-script.webp)
```
Comment (tianzhou) [inline content/blog/the-sql-editor-for-developers-and-dbas.md:49]:
```suggestion
![large-script](/static/blog/the-sql-editor-for-developers-and-dbas/large-script.webp)
```

File: content/blog/the-sql-editor-for-developers-and-dbas.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+![access-control](/static/blog/the-sql-editor-for-developers-and-dbasaccess-control.webp)
+
+### Data Anonymization
+
+When developers query tables containing sensitive columns in SQL Editor, the result of these columns will display as "**\*\***". You can learn the configuration details from the document [Anonymize Data](https://www.bytebase.com/docs/administration/anonymize-data).
+![anonymize-data](/static/blog/the-sql-editor-for-developers-and-dbasanonymize-data.webp)
+
+### Collaborative Sheets
+
+For commonly-used SQL statements, you can save them as sheets and set up a public SQL script library in your team. For large SQL scripts (up to 100M), you can use them in issues after uploading them as sheets (see the screenshot below). You can explore the document [Manage SQL Scripts with Sheet](https://www.bytebase.com/docs/sql-editor/manage-sql-scripts) to get additional details.
```
Comment (tianzhou) [inline content/blog/the-sql-editor-for-developers-and-dbas.md:48]:
```suggestion
For commonly-used SQL statements, you can save them as sheets and set up a public SQL script library in your team. For large SQL scripts (up to 100M), you can use them in issues after uploading them as sheets (see the screenshot below). You can explore the document [Manage SQL Scripts with Sheet](/docs/sql-editor/manage-sql-scripts) to get additional details.
```

File: content/blog/the-sql-editor-for-developers-and-dbas.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+![admin-mode](/static/blog/the-sql-editor-for-developers-and-dbasadmin-mode.webp)
+
+### Database-based Access Control
+
+When users want to allow developers to access some databases in the production environment, they can mark the production environment as a protected environment and configure an allowlist. You can get more details from the document [Database Access Control](https://www.bytebase.com/docs/administration/database-access-control).
+![access-control](/static/blog/the-sql-editor-for-developers-and-dbasaccess-control.webp)
+
+### Data Anonymization
+
+When developers query tables containing sensitive columns in SQL Editor, the result of these columns will display as "**\*\***". You can learn the configuration details from the document [Anonymize Data](https://www.bytebase.com/docs/administration/anonymize-data).
```
Comment (tianzhou) [inline content/blog/the-sql-editor-for-developers-and-dbas.md:43]:
```suggestion
When developers query tables containing sensitive columns in SQL Editor, the result of these columns will display as "**\*\***". You can learn the configuration details from the document [Anonymize Data](/docs/administration/anonymize-data).
```

File: content/blog/the-sql-editor-for-developers-and-dbas.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+- Good user experience: This release presents SQL Editor with a fresh look and improves the code auto-completion.
+
+### Admin Mode
+
+When users want to connect to databases to run admin commands, they can do it in SQL Editor Admin mode with a DBA account like the screenshot below. You can take a look at the document [Admin Mode](/docs/sql-editor/admin-mode) to learn more details.
+![admin-mode](/static/blog/the-sql-editor-for-developers-and-dbasadmin-mode.webp)
+
+### Database-based Access Control
+
+When users want to allow developers to access some databases in the production environment, they can mark the production environment as a protected environment and configure an allowlist. You can get more details from the document [Database Access Control](https://www.bytebase.com/docs/administration/database-access-control).
```
Comment (tianzhou) [inline content/blog/the-sql-editor-for-developers-and-dbas.md:38]:
```suggestion
When users want to allow developers to access some databases in the production environment, they can mark the production environment as a protected environment and configure an allowlist. You can get more details from the document [Database Access Control](/docs/administration/database-access-control).
```

**Review by tianzhou — CHANGES_REQUESTED** [review]

**Comment by Candybase** [comment]
> I can't view the images, don't know why ...

Fixed

File: content/blog/the-sql-editor-for-developers-and-dbas.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+![audit-log](/static/blog/the-sql-editor-for-developers-and-dbas/audit-log.webp)
+
+### Collaborative Sheets
+
+For commonly-used SQL statements, you can save them as sheets and set up a public SQL script library in your team. For large SQL scripts (up to 100M), you can use them in issues after uploading them as sheets (see the screenshot below). You can explore the document [Manage SQL Scripts with Sheet](/docs/sql-editor/manage-sql-scripts) to get additional details.
+![large-script](/static/blog/the-sql-editor-for-developers-and-dbas/large-script.webp)
+
+## Try it Out
+
+With the new SQL Editor in Bytebase Console, you can run database admin commands in Admin mode, anonymize sensitive data when executing queries, configure a database allowlist to manage database access permission, and save commonly-used SQL scripts to collaborate with team members. Besides that, you can view all queries in SQL Editor with the audit logs. We hope you can find the SQL Editor to be a useful tool.
```
Comment (tianzhou) [inline content/blog/the-sql-editor-for-developers-and-dbas.md:59]:
```suggestion
With the new SQL Editor in Bytebase Console, you can run database admin commands in Admin mode, anonymize sensitive data when fetching query results, configure a database allowlist to manage database access permission, and save commonly-used SQL scripts to collaborate with team members. Besides that, you can inspect all queries in SQL Editor via audit logs.
```

File: content/blog/the-sql-editor-for-developers-and-dbas.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+
+If you have any comments or questions, don’t hesitate to let us know. You can join our discord at https://discord.gg/Fac9nmZ95j to learn more about Bytebase.
```
Comment (tianzhou) [inline content/blog/the-sql-editor-for-developers-and-dbas.md:61]:
```suggestion
If you have any comments or questions, don’t hesitate to let us know. You can join our [discord](https://discord.gg/Fac9nmZ95j) to learn more about Bytebase.
```

**Review by tianzhou — APPROVED** [review]
LGTM

**Review by adela-bytebase — APPROVED** [review]
LGTM

**Pull request merged**